packages/vc-verification/src/resolver/data-aggregator.ts

```diff
@@ -0,0 +1,54 @@
+import {
+  CachedDIDDocument,
+  CachedRoleCredential,
+  CachedRoleDefinition,
+} from '../models';
+
+/**
+ * An Interface for Data aggreator to cache responses from Blockchain and SSI-Hub
+ */
+export interface IDataAggregator {
```

Comment:
Is it supposed to be different implementations of IDataAggregator?

Reply:
Yeah, there would be different implementation of IDataAggregator. So for the case in `ew-credentials` it is just the data coming from blockchain. While in `passport-did-auth` the data would be coming from both blockchain and SSI-Hub.

Reply:
What is expected difference would be between their implementations. Is `passport-did-auth` version expected to do something else except getting and setting? 
Also name `EVMDataAggregator` is quite vague from my perspective

Reply:
The data type that is being stored would create difference. Since data returned from SSI-Hub are different to what comes from blockchain. 

>Also name EVMDataAggregator is quite vague from my perspective

Yeah, I didn't find any other relevant naming for this.

Reply:
I agree with @JGiter 's point that `EVMDataAggregator` is quite vague. To me, this is because it is violating the single-responsability-principle and so it is hard to find a specific name for it. I recommend splitting the cache up into a cache for each type of cached data.

Reply:
The `IntermediaryDataCache` doens't point out what is specific in this implementation of `IDataCache`. You argued that interface introduced because data from ssi-hub differs from data from chain. Could you please specify which exact data do you mean. I thought that ssi-hub implements interfaces defined in ew-credentials